src/libs/actions/Policy.js

```diff
@@ -822,7 +819,7 @@ function createWorkspace() {
     },
     {
         optimisticData: [{
-            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            onyxMethod: CONST.ONYX.METHOD.SET,
```

Comment:
NAB: I would also use `SET` for every key on the optimistic data so that it's available right away. Maybe it's fine like this, but there's no risk of overwriting existing data with `Onyx.set` because all of this data is new, so why not?

Reply:
Yeah this is a good idea, it should be fine to just SET if we can remove the makeRequestWithSideEffects

Reply:
done!